csharp/authenticate/authenticate/Program.cs

```diff
@@ -13,6 +14,7 @@ static void Main(string[] args)
             string username = "";
             string password = "";
             string path = "database.json";
+            var hasher = new BCrypt.Net.BCrypt();
```

Comment:
Not sure why there is even a constructor since the other methods I used are static. Probably worth investigating.